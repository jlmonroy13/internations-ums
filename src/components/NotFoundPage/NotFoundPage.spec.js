import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage from './NotFoundPage';

describe('<NotFoundPage />', () => {
  it('should render the not found page component', () => {
    const wrapper = renderer.create(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
