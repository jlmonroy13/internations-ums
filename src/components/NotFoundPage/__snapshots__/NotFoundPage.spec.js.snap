// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should render the not found page component 1`] = `
<div>
  <h4>
    404 Page Not Found
  </h4>
  <a
    onClick={[Function]}
    style={Object {}}
  >
     Go back to homepage 
  </a>
</div>
`;
